.github/actions/upload-artifacts/action.yml

```diff
@@ -22,7 +22,7 @@ runs:
       shell: bash
       run: |
         mkdir -p ${{ inputs.artifacts-dir }} && \
-        mv ${{ inputs.target-file }} ${{ inputs.artifacts-dir }}
+        cp ${{ inputs.target-file }} ${{ inputs.artifacts-dir }}
```

Comment:
Why do we change mv to cp?

Reply:
Because the greptime binary built by docker in the previous step is under root user privilege while the command executor here is user "runner" if we are using github hosted runners. If mv, it has permission denied problem.